server/aws-lsp-codewhisperer/src/language-server/chat/chatController.test.ts

```diff
@@ -59,7 +59,7 @@ describe('ChatController', () => {
     const expectedCompleteChatResult: ChatResult = {
         messageId: mockMessageId,
         body: 'Hello World!',
-        canBeVoted: true,
+        canBeVoted: false,
```

Comment:
actually you probably don't need to change the normal chat controller, you only need to change the agentic chat controller ones

Reply:
yes, the change should not affect the base chat